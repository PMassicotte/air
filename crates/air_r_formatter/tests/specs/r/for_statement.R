for(x in y) x + y

for(a_really_long_argument_name_to_break_on in another_really_long_variable_name_here) 1

for # dangling1
  ( # dangling2
    a in 1
  ) # dangling3
  a
