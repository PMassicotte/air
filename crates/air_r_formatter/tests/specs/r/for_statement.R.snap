---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/for_statement.R
---
# Input

```R
for(x in y) x + y

for(a_really_long_argument_name_to_break_on in another_really_long_variable_name_here) 1

for # dangling1
  ( # dangling2
    a in 1
  ) # dangling3
  a

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
-----

```R
for (x in y) x + y

for (
	a_really_long_argument_name_to_break_on
	in
	another_really_long_variable_name_here
) 1

for (
	# dangling1
	# dangling2
	a
	in
	1 # dangling3
) a
```
